README.md

```diff
@@ -16,7 +16,7 @@ The following command can install dependencies for you.
 mvn package
 ```
 
```

Comment:
when  execute `mvn package`  , first need to install `maven` on your computer  whether need to give a tip?